scripts/examples/gn_silabs_example.sh

```diff
@@ -192,6 +198,10 @@ else
                 optArgs+="chip_enable_icd_server=true chip_openthread_ftd=false "
                 shift
                 ;;
+            --ssed)
```

Comment:
do we need the prefix for 1 arg?  or should this enable MTD also?

Reply:
removed